core/common/src/main/java/alluxio/wire/FileInfo.java

```diff
@@ -612,8 +625,8 @@ public boolean equals(Object o) {
         && mFileBlockInfos.equals(that.mFileBlockInfos) && mTtlAction == that.mTtlAction
         && mMountId == that.mMountId && mInAlluxioPercentage == that.mInAlluxioPercentage
         && mUfsFingerprint.equals(that.mUfsFingerprint)
-        && mAclEntries.equals(that.mAclEntries)
-        && mDefaultAclEntries.equals(that.mDefaultAclEntries);
+        && mAcl.equals(that.mAcl)
+        && mDefaultAcl.equals(that.mDefaultAcl);
```

Comment:
what if this is `null`?

Reply:
made the decision to not allow them to be `null`